assets/main.js

```diff
@@ -26,3 +31,56 @@ document.addEventListener('DOMContentLoaded', () => {
         }
     });
 });
+
+function startEventSource() {
+	if (queryStreamEvent) return;
+
+	if (!window.location.pathname.startsWith("/querylog")) return;
+
+	const query_logs = document.getElementById("query-logs");
+
+	queryStreamEvent = new EventSource(`/api/queries`);
+
+	queryStreamEvent.onmessage = ({ data }) => {
+		try {
+		    const payload = JSON.parse(data);
+                    let row = query_logs.insertRow(-1);
```

Comment:
we should add to top of the table, and if there are more than X (let's say 50) elements in the table, remove the last one.